backend/zeno_backend/processing/metrics/map.py

```diff
@@ -24,32 +24,34 @@ def count(
     Returns:
         GroupMetric: count of datapoints matching the specified filter.
     """
-    with Database() as db:
-        num_total = db.execute_return(
-            sql.SQL("SELECT COUNT(*) FROM {}").format(sql.Identifier(project))
-            if filter is None
-            else sql.SQL("SELECT COUNT(*) FROM {} WHERE ").format(
-                sql.Identifier(project)
+    async with db_pool.connection() as db:
+        async with db.cursor() as cur:
```

Comment:
We're repeating this in every function. Should we move it to a function or something? Maybe not worth would make one line out of two....

Reply:
Yeah thought about this but some also use the connection itself and stuff, idk felt like it's easier just the raw way